fourmat/lint.py

```diff
@@ -36,12 +102,17 @@ def copy_configuration(override=False):
 
 
 def black(paths, *, check=False):
+    # Do not override extend-exclude if the user has specified it in their black configuration
+    exclude_arg = (
+        ()
+        if "extend_exclude" in (PY_PROJECT_CONFIG.black or {})
+        else ("--extend-exclude", SNAPSHOT_REGEX)
```

Comment:
yes better. +1 also for switching to extend-exclude. I actually think we can/should probably drop the snapshot regex as it feels too opinionated to me